packages/components/src/h-stack/README.md

```diff
@@ -83,7 +89,12 @@ Determines if children should wrap.
 When a `Spacer` is used within an `HStack`, the `Spacer` adaptively expands to take up the remaining space.
 
 ```jsx
-import { HStack, Spacer, Text, View } from '@wordpress/components/ui';
+import {
+	__experimentalHStack as HStack,
+	__experimentalSpacer as Spacer,
```

Comment:
There is no `Spacer` in `@wordpress/components` yet. Should we replace it with something else in this README file?